### ISSUE TITLE
Thank you for your work. I'd appreciate it if you could provide a trained model for me to test.

### ISSUE THREAD

**Issue by wangzhijiang110** [edshkim98/3Dteeth_modelling#1]

**Comment by edshkim98** [comment]
Hi @wangzhijiang110  thank you for visiting my repository.
I am still very actively researching on this project and my last update which was almost a month ago is almost just a plain DGCNN with a combination of different loss functions. However, I have been adding some of more functions on the model, so it has changed a bit since the last update. Hence, I am not sure whether I have the exact pre-trained model.
This means I will be pushing a newer version soon and will also consider including the pre-trained model as well!
By the way, 'pts' and 'seg' are the text files for storing x,y,z and label.

**Comment by wangzhijiang110** [comment]
Thank you for your reply. One more question, please.If I have my own STL file.How to label data.

**Comment by edshkim98** [comment]
First, when you open STL files using open3d or trimesh, it will return arrays with vertices and faces because my dataset comes
in mesh form. Although, I am currently also using faces as well, initially I only used vertices x,y,z and a label.
These values for vertices and labels will both be saved in pts and seg files with the same name.
pts and seg are basically txt files but in different formats.
Hence, it will look as follow:
1.pts file (points)
x1 y1 z1
x2 y2 z2

1.seg file (label)
1 
0 and so on

I hope this helps you.
Thank you.

**Comment by wangzhijiang110** [comment]
Thank you, your reply was very helpful to me.Since marking data is a laborious job, could you please give me one data you have already marked?Hope you can give me a tooth for all the files.(One tooth will do)I would like to refer to it. As a thank you, I will mark up the 20 cases I have and send them to you.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "edshkim98/3Dteeth_modelling"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年11月5日(星期四) 晚上9:19
收件人:&nbsp;"edshkim98/3Dteeth_modelling"<3Dteeth_modelling@noreply.github.com&gt;;
抄送:&nbsp;"一念永恒"<895067048@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [edshkim98/3Dteeth_modelling] Thank you for your work. I'd appreciate it if you could provide a trained model for me to test. (#1)





 
First, when you open STL files using open3d or trimesh, it will return arrays with vertices and faces because my dataset comes
 in mesh form. Although, I am currently also using faces as well, initially I only used vertices x,y,z and a label.
 These values for vertices and labels will both be saved in pts and seg files with the same name.
 pts and seg are basically txt files but in different formats.
 Hence, it will look as follow:
 1.pts file (points)
 x1 y1 z1
 x2 y2 z2
 
1.seg file (label)
 1
 0 and so on
 
I hope this helps you.
 Thank you.

**Comment by edshkim98** [comment]
Hi, I can upload a file for you, but it will be pre-processed data due to not being able to disclose raw data. This means the data will have 20k sampled point cloud with labels saved in pts and seg files.
I will try to push it tonight, so you can come back later to check.
Thanks
